src/hooks/useMetaMask.ts

```diff
@@ -46,7 +46,7 @@ export function metaMaskRequests(info: MetaMaskInfo, type: string, account?:stri
                   symbol: 'FUSE',
                   decimals: 18
               },
-              rpcUrls: [process.env.REACT_APP_FUSE_RPC ?? 'https://rpc.fuse.io'],
+              rpcUrls: [process.env.REACT_APP_FUSE_RPC],
```

Comment:
this fallback was added because it fails to load without it cause REACT_APP_FUSE_RPC is potentially undefined